src/components/Button.js

```diff
@@ -135,30 +136,47 @@ const Button = React.createClass({
         ':hover': {
           color: StyleConstants.adjustColor(this.props.primaryColor, -8),
           fill: StyleConstants.adjustColor(this.props.primaryColor, -8),
-          transition: 'all .2s ease-in'
+          transition: 'all .2s ease-in',
+          borderColor: StyleConstants.Colors.FOG,
+          borderRadius: 2,
+          borderWidth: 1
```

Comment:
To clarify. I believe the error is only thrown if you mix shorthand/longhand related styles. For example, mixing `border` with `border-width` will throw an error. Mixing `border` with `width` would not throw an error.